ajax/libs/angular-parse/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "angular-parse",
+  "filename": "angular-parse.js",
+  "version": "0.2.3",
+  "description": "Angular wrapper for Parse.com JavaScript SDK.",
+  "license": "MIT",
+  "author": "<ivnivnch@gmail.com>",
```

Comment:
Though we still does't use `author` for important uses. It seems a little strange to only use email here, how about adding the author's name? `ivan <ivnivnch@gmail.com>`